packages/design-system/src/components/web-components/__snapshots__/ds-usa-banner.test.tsx.snap

```diff
@@ -1,105 +1,218 @@
+              fill="#FFF"
+            />
+          </g>
+        </svg>
+        <p
+          class="ds-c-usa-banner__header-text"
+        >
+          An official website of the United States government
+        </p>
+        <p
+          class="ds-c-usa-banner__action ds-u-align-items--center ds-u-md-display--none ds-u-display--flex"
+        >
+          Here's how you know
+          <svg
+            aria-hidden="true"
+            class="ds-c-icon ds-c-icon--arrow ds-c-icon--arrow-down ds-c-usa-banner__action-icon"
+            direction="down"
+            viewBox="0 0 320 512"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
+              fill="currentColor"
+            />
+          </svg>
+        </p>
+        <button
+          aria-controls="static-id__panel"
+          aria-expanded="false"
+          class="ds-c-usa-banner__button"
+          type="button"
+        >
+          <span
+            class="ds-c-usa-banner__button-text ds-u-md-display--flex ds-u-align-items--center"
+          >
+            Here's how you know
+            <svg
+              aria-hidden="true"
+              class="ds-c-icon ds-c-icon--arrow ds-c-icon--arrow-down ds-c-usa-banner__action-icon"
+              direction="down"
+              viewBox="0 0 320 512"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
+                fill="currentColor"
+              />
+            </svg>
+          </span>
+          <span
+            class="ds-c-usa-banner__button-icon-container"
+            hidden=""
+          >
+            <svg
+              aria-hidden="true"
+              class="ds-c-icon ds-c-icon--close ds-c-icon--close-thin ds-c-usa-banner__button-icon"
+              viewBox="-2 -2 18 18"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M0 13.0332964L13.0332964 0M13.0332964 13.0332964L0 0"
+                fill="none"
+                stroke="currentColor"
+                stroke-linecap="round"
+                stroke-width="2"
+              />
+            </svg>
+          </span>
+        </button>
+      </header>
+      <div
+        class="ds-c-usa-banner__guidance [ ds-u-leading--base ds-u-font-size--base ]"
```

Comment:
Huh, this class name is a little weird 🤔

Reply:
oh, i'd grouped those specifically in usa banner using brackets (the brackets don't impact css like this). there's a comment in the component saying these specific classes are used to hard code font treatment across themes. this is the only component like this in our DS so i was trying to call out those 2 utility classes were together and important for that reason.